src/internal/grid/styles.scss

```diff
@@ -5,10 +5,14 @@ $grid-col-width: minmax(0, 1fr);
 .grid {
   display: grid;
   /* Matches GRID_GAP constant used for calculations. */
-  /* TODO: consider using different gaps for comfortable/compact which would require a mode observer. */
-  gap: 16px;
+  gap: 20px;
```

Comment:
Same here. Should we introduce a design token for this?

Reply:
Good idea - I'll check if we can already use the design tokens for the gaps. For the height I don't think that makes much sense

Reply:
Actually, introducing the design tokens does not make it easier. We do need to have these values in JS and those need to be consistent. The imported design tokens is giving a variable name, not a value. It is possible to read the computed values from CSS for both `gap` and `grid-auto-rows` but that would require the same mutation observer. I also tried it this way and found out that the variable is not being set synchronously - when reading it after the new mode is confirmed from the observer does not result in a matching value unless a delay is added.

Reply:
Good explanation, resolving